packages/consts/src/consts.ts

```diff
@@ -111,7 +116,25 @@ export const USERNAME = {
 
     // Regex matching a potentially allowed username. The numbers must match MIN and MAX!
     // Note that username must also pass isForbiddenUser() test to be allowed!
-    REGEX: /^[a-zA-Z0-9_.-]{3,30}$/,
+    REGEX: /^[a-zA-Z0-9_-]{3,30}$/,
+};
+
+export const EMAIL = {
+    MIN_LENGTH: 3,
+    MAX_LENGTH: 254, // see https://www.rfc-editor.org/errata_search.php?rfc=3696&eid=1690
```

Comment:
Moved from apify-core as we had quite a bit of inconsistencies in a ways we handle email restrictions across the org.